examples/AdWords/v201607/CampaignManagement/AddCompleteCampaignsUsingBatchJob.php

```diff
@@ -112,9 +112,10 @@ function AddCompleteCampaignUsingBatchJobExample(AdWordsUser $user) {
         $batchJob->status);
 
     $pollAttempts++;
-    if ($batchJob->status !== 'ACTIVE' &&
-        $batchJob->status !== 'AWAITING_FILE' &&
-        $batchJob->status !== 'CANCELING') {
+    if (!in_array($batchJob->status, array(
```

Comment:
The main benefit of in_array() is the readability (over the micro-optimizations already explained in main description of the PR) and set an example for future coders that will write similar code based in the adwords examples. Currenty the BatchJobStatus has <10 status, and in this check probably we are only using 3 values of checking and 3 conditions to avoid, but when other less experienced coders write examples or production code, you could find really spaguetti code because they just copy and paste from the examples.

In other side, if this lib deprecates in experimental branch PHP 5.3 and PHP 5.4 (or only PHP 5.3) since are EOL, we could use a better sintax for arrays like [ ] which improves a lot the readability IMHO.

    if (!in_array($batchJob->status, 
        [
            BatchJobStatus::ACTIVE,     
            BatchJobStatus::AWAITING_FILE, 
            BatchJobStatus::CANCELING
        ])) {

Also I read again the [Contributing spec](https://github.com/googleads/googleads-php-lib/blob/master/CONTRIBUTING.md) and I could not find anything related with coding standards, so I am not sure about the proper identation. It would be useful too for future PR, if you add some bot that checks the coding standards (like [this](https://github.com/markstory/lint-review) or others that other open source projects are using it)

I am not sure if you are refering to use the brackets in [Stroustrup](https://en.wikipedia.org/wiki/Indent_style#Variant:_Stroustrup) style (personally I like more the [K&R style](https://en.wikipedia.org/wiki/Indent_style#K.26R_style), but no problem changing it that)

Let me know the needed after you read all this and I will update the PR as you finally decide.